<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Astro</class>
 <widget class="QWidget" name="Astro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>406</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="1">
    <widget class="QWidget" name="doppler_widget" native="true">
     <property name="styleSheet">
      <string notr="true">* {
	font-weight: normal;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Frequency above nominal band edge</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="kHzSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="suffix">
            <string>  kHz</string>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="HzSpinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="suffix">
            <string>   Hz</string>
           </property>
           <property name="minimum">
            <number>-2000</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Doppler tracking</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbFullTrack">
           <property name="text">
            <string>Full Doppler to DX Grid</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbConstFreqOnMoon">
           <property name="text">
            <string>Constant frequency on Moon</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNoDoppler">
           <property name="text">
            <string>None</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Transceiver step size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="rb1Hz">
           <property name="text">
            <string>1 Hz</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb10Hz">
           <property name="text">
            <string>10 Hz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb100Hz">
           <property name="text">
            <string>100 Hz</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Tx audio tracking</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="cbTxAudioTrack">
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="text_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
	font-family: Courier;
	font-size: 12pt;
	font-weight: bold;
}</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Astro Data</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="cbDopplerTracking">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Doppler tracking</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
